src/appengine/handlers/cron/ml_train.py

```diff
@@ -28,17 +28,22 @@ class Handler(base_handler.Handler):
   def get(self):
     """Handle a GET request."""
     for job in data_types.Job.query():
+
       if not utils.string_is_true(
           job.get_environment().get('USE_CORPUS_FOR_ML')):
         continue
 
+      task_name = 'gradientfuzz' if utils.string_is_true(
```

Comment:
given the length of this condition, this would bve more readable as:

```python
if utils.string_is_true(...):
  task_name = 'gradientfuzz'
else:
  task_name = 'ml_train'
```

Reply:
Got it -- fixed :)